dlt/extract/decorators.py

```diff
@@ -449,6 +455,15 @@ def make_resource(
             schema_contract=schema_contract,
             table_format=table_format,
         )
+
+        # If custom nesting level was specified then
+        # we need to add it to table hints so that
+        # later in normalizer dlt/common/normalizers/json/relational.py
+        # we can override max_nesting level for the given table
+        if max_table_nesting is not None:
```

Comment:
do we have `max_table_nesting` available as property of dlt source? I think so. so let's do the same for resource. will be handy for users

Reply:
Done